### PR TITLE
Add lsst-ts-xml recipe

### PR DESCRIPTION
**Pull request by couger01**

### CODE REVIEW

**Comment by rbovill** [comment]
I have been, and ever shall be, a maintainer of the ts_xml package.

**Comment by tribeiro** [comment]
Agree to be a maintainer.

**Review by beckermr — CHANGES_REQUESTED** [review]
Please remove the comments.

File: recipes/lsst-ts-xml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lsst-ts/ts_xml/archive/v{{ version }}.tar.gz
+  sha256: 04a8c8053c1c26180b689c61ab470de8117f6ac243158f17b17aa1a288dfc5fe
+
+build:
+  skip: True  # [win, py<310]
```
Comment (beckermr) [inline recipes/lsst-ts-xml/meta.yaml:13]: This recipe should be noarch python with a constraint on python in the recipe directly.

**Review by beckermr — CHANGES_REQUESTED** [review]

File: recipes/lsst-ts-xml/meta.yaml (hunk 1 above)
Comment (couger01) [reply recipes/lsst-ts-xml/meta.yaml:13]: Except it didn't build successfully with Windows and therefore when I set the skip: true [win], the linter said that having the skip meant that noarch:python couldn't be used.

Comment (beckermr) [reply recipes/lsst-ts-xml/meta.yaml:13]: You can ignore the windows failure. Noarch python only builds on Linux.

File: recipes/lsst-ts-xml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools_scm
+  run:
+    - python
+    - astropy
```
Comment (beckermr) [inline recipes/lsst-ts-xml/meta.yaml:24]:
```suggestion
    - astropy
    - _unix
```
This will ensure it only installs on osx and Linux.

**Review by beckermr — COMMENTED** [review]
This change will ensure people cannot install this on windows.

File: recipes/lsst-ts-xml/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  sha256: 04a8c8053c1c26180b689c61ab470de8117f6ac243158f17b17aa1a288dfc5fe
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
```
Comment (beckermr) [inline recipes/lsst-ts-xml/meta.yaml:19]:
```suggestion
    - python >=3.10
```

File: recipes/lsst-ts-xml/meta.yaml (hunk 2 above)
Comment (couger01) [reply recipes/lsst-ts-xml/meta.yaml:24]: It has to be with a double underscore.

**Pull request merged**